AWSAppSyncClient/Internal/AWSAppSyncClientInfo.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+
+import Foundation
+
+public protocol AWSAppSyncClientInfoProviding {
+    var apiUrl: String { get }
+    var region: String { get }
+    var authType: String { get }
+    var apiKey: String { get }
+}
+
+/**
+ * Configuration for AWSAppSyncClient as read from `awsconfiguration.json`
+ */
+public class AWSAppSyncClientInfo: AWSAppSyncClientInfoProviding {
```

Comment:
It would be great if you take changes from https://github.com/awslabs/aws-mobile-appsync-sdk-ios/blob/25935a273ab7092d6a49c1868d9099379033daab/AWSAppSyncClient/AWSAppSyncClientInfo.swift.

This removed force unwraps, which are really bad thing.

Reply:
Thanks for the suggestions, I will incorporate that.